styleguide/_themes/global/scss/tables.scss

```diff
@@ -108,3 +108,97 @@ $table-padding: 20px 5px;
     }
   }
 }
+
+//Compare Tables
+.compareTable {
+  background-color: transparent;
+  border-collapse: collapse;
+  border-spacing: 1px;
+  display: table;
+  font-size: 13px;
+  min-height: 100px;
+  overflow: hidden;
+  table-layout: fixed;
+  text-align: left;
+  width: 100%;
+}
+
+.compareTable-thead {
+  background-color: $teal-medium;
+  color: $white;
+  display: table-header-group;
+  font-weight: bold;
+  text-transform: uppercase;
+  width: 100%;
+}
+
+.compareTable-theadRow {
+  display: table-row;
+}
+
+.compareTable-theadCell {
+  display: table-cell;
+  overflow: hidden;
+  padding: 20px 12px;
+}
+
+.compareTable-theadCell-long {
+  width: 70%;
+}
+
+.compareTable-body {
+  display: table-row-group;
+}
+
+.compareTable-bodyRow {
+  display: table-row;
+
+  &:nth-child(even) {
+    background-color: transparent;
+  }
+
+  &:nth-child(odd) {
+    background-color: $gray-lightest;
+  }
+}
+
+.compareTable-bodyCell {
+  display: table-cell;
+  overflow-x: auto;
+  padding: 12px;
+  vertical-align: top;
+
+  .fa-check {
+    margin-left: 35px;
+    margin-top: 20px;
+  }
+}
+
+.compareTable-listItem {
+  font-size: 11px;
+}
+
+@media (max-width: $screen-sm) {
```

Comment:
This should either be `$screen-sm-max` or `$screen-xs-max` depending on where you're trying to break.